docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -276,3 +276,141 @@ The POST request body should be in JSON format with the following structure:
 
 * `400 Bad Request` - some parameters in request body were missed or wrong.
 * `401 Unauthorized` - projectID verification error.
+
+## Convertation
+
+Endpoints to make token convertation transactions
+
+### List of tokens available for convertation
+
+`GET /v1/convert/tokens`
```

Comment:
@geekbrother I think we will need two more enpoint for listing tokens:

- `Popular Tokens` - Most popular tokens to let user select easily while converting
- `Search Tokens`- Search tokens with a query

Reply:
Probably this can be achieved by filtering at the single endpoint, as the response schema is the same.

> Popular Tokens - Most popular tokens to let users select easily while converting

We need to get or maintain the list of popular tokens to get and probably just use the filtering when request the token list.

> Search Tokens- Search tokens with a query

At the moment we have a [filter by the chainID](https://github.com/WalletConnect/walletconnect-specs/pull/207/files#diff-f64bf5c2b17c9c6d32ae9d85d9a1ce5a4aa07cb680c6f4085de94d3c4915b9d9R291). What other filters you suggest to add @enesozturk? Ww can use filtering by token symbol or address for example.